inspectit-oce-agent/src/system-test/java/rocks/inspectit/oce/instrumentation/special/ExecutorContextPropagationTest.java

```diff
@@ -12,15 +12,15 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static rocks.inspectit.oce.TestUtils.waitForInstrumentationToComplete;
 
 public class ExecutorContextPropagationTest {
 
     private static final Tagger tagger = Tags.getTagger();
 
     @BeforeAll
-    static void waitForInstrumentation() throws Exception {
```

Comment:
Would it make sense that we create an abstract base class for all our system-tests which contains stuff like this (waiting for instrumentation)?
Then we just need to extend this class instead of having a separate before-all in each test.